src/main/java/sirpotato/Parser.java

```diff
@@ -0,0 +1,111 @@
+package sirpotato;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+
+/**
+ * Contains methods to parse user input and validate user input
+ */
+class Parser {
+
+    /**
+     * Validates the date that user inputs
+     * If not in dd-MM-yyyy format, will throw an exception
```

Comment:
Shouldn't there be an empty line between the description and parameters section in javadoc?
I've noticed that this occurs for other Javadoc comments as well.